sparkler-core/sparkler-ui/pom.xml

```diff
@@ -46,6 +46,7 @@
     </dependencies>
 
     <build>
+        <finalName>${project.artifactId}</finalName>
```

Comment:
Can this not be moved into the `<properties>` section?

Reply:
This was necessary for banana to work with our sparkler deployment. Since we're using kibana, we don't need this change and are going to revert it.